rudder-web/src/main/webapp/style/newTheme/newTheme.css

```diff
@@ -0,0 +1,590 @@
+/*
```

Comment:
@RaphaelGauthier Can you move that file in another directory ? /style/rudder ?  use 'git mv' command to move it!